### PR TITLE
Fix lastEntry match is not supporting directories #71

### PR DESCRIPTION
**Pull request by yakirzana**
Fix lastEntry match is not supporting directories #71

### CODE REVIEW

File: FileStreamRotator.js [hunk 1]
```diff
@@ -480,10 +480,12 @@ FileStreamRotator.getStream = function (options) {
         var f = null;
         if(auditLog && auditLog.files && auditLog.files instanceof Array && auditLog.files.length > 0){
             var lastEntry = auditLog.files[auditLog.files.length - 1].name;
-            if(lastEntry.match(t_log)){
-                var lastCount = lastEntry.match(t_log + "\\.(\\d+)");
+            var lastEntryWithoutPath = lastEntry.split("\\").pop();
+            var t_logWithoutPath = t_log.split("\\").pop();
```
Comment (mattberther) [inline FileStreamRotator.js:484]: @yakirzana I wonder if you want to substitute `"\\"` with `path.sep` to return the platform-specific file separator.

Comment (yakirzana) [reply FileStreamRotator.js]: Yes, sure.

**Comment by yakirzana** [comment]
@rogerc There is an option to check this?

**Comment by rogerc** [comment]
this appears to be a Windows issue related to its path separator as its not being escaped in the regexp. I'll be working in a new major release over the coming months and will support paths in windows environments.

**Pull request closed without merging**